From 28f252a04151a439b62bb929a3dd2d771f7b91dd Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Tue, 9 Sep 2014 12:46:17 +0200
Subject: [PATCH] systemd: remove unsupported option ControlGroup=

---
 rtkit-daemon.service.in | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/rtkit-daemon.service.in b/rtkit-daemon.service.in
index 3dfefa6..d0dc786 100644
--- a/rtkit-daemon.service.in
+++ b/rtkit-daemon.service.in
@@ -27,9 +27,5 @@ CapabilityBoundingSet=CAP_SYS_NICE CAP_DAC_READ_SEARCH CAP_SYS_PTRACE CAP_SYS_CH
 PrivateTmp=yes
 PrivateNetwork=yes
 
-# Work around the fact that the Linux currently doesn't assign any RT
-# budget to CPU control groups that have none configured explicitly
-ControlGroup=cpu:/
-
 [Install]
 WantedBy=graphical.target
-- 
1.8.3.1

